### PR TITLE
Hot fix for issue #54, skipping tree calc if no core gene MSA

### PR DESCRIPTION
**Pull request by hoelzer**

### CODE REVIEW

**Comment by hoelzer** [comment]
We will skip the aln filtering and tree calculation if there is no single MSA with **all** input species. This happend in this example #54 

However, it would be better to calculate a tree from other MSAs in such a case. E.g. when there is no single MSA with all 28 input species, check for MSAs with 27 species and so on. 

For now, this should fix the error message at least and finish the workflow (with tree).

**Comment by hoelzer** [comment]
I am now considering relaxing the “core genes” cutoff in RIBAP. In the current implementation, we’re simply defining all genes we find in all input species (e.g. 28/28) as the core. And use them in the core gene MSA and phylogeny.

But I see now that this can be too restrictive, so I am considering a user-defined percentage cutoff for the species needed to define a core gene (as other pipelines do as well). The default could still be 100%, but yweou could lower it then to e.g., 95% (26/28 species required to define a core gene). It might also make sense because we designed RIBAP specifically to look into more diverged genomes.

**Comment by hoelzer** [comment]
@klamkiew what do you think? I am testing it right now, but seems to work. Can we run into problems because we feed MSAs with different sets of species into iqtree? I dont think so, the tool should be able to handle that.

**Comment by klamkiew** [comment]
I don't know whether IQ-Tree handles this... One option might be to "fill" the MSA with gaps for a species that does not have this gene. But let's wait for your test first. 
I will have a more throughout look later.

File: modules/filter_alignment.nf
Comment (klamkiew) [inline modules/filter_alignment.nf:1]: I think the `if`-condition will spam the log file of nextflow. We could add
```
if ls ./ | grep ... > /dev/null; then
[...]
fi
```

Comment (hoelzer) [reply modules/filter_alignment.nf:1]: good point, I am adding that. Test is still running for 28 species input where there is no RIBAP group with all 28 members. So I am setting the new cutoff to 0.95 which will consider MSAs with 27 species. The tree calculation is still running...

**Pull request merged**